<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fullScreenForm</class>
 <widget class="QDialog" name="fullScreenForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>521</height>
   </rect>
  </property>
  <property name="tabletTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QPushButton" name="pbtn_prePage">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>232</y>
     <width>62</width>
     <height>56</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="hg_sane_rsc.qrc">
     <normaloff>:/image_rsc/ButtonIco/Active/btnPrevious.png</normaloff>:/image_rsc/ButtonIco/Active/btnPrevious.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>761</width>
     <height>501</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="dragMode">
    <enum>QGraphicsView::ScrollHandDrag</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtn_nextPage">
   <property name="geometry">
    <rect>
     <x>703</x>
     <y>232</y>
     <width>62</width>
     <height>56</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="hg_sane_rsc.qrc">
     <normaloff>:/image_rsc/ButtonIco/Active/btnNext.png</normaloff>:/image_rsc/ButtonIco/Active/btnNext.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <zorder>graphicsView</zorder>
  <zorder>pbtn_prePage</zorder>
  <zorder>pbtn_nextPage</zorder>
 </widget>
 <resources>
  <include location="hg_sane_rsc.qrc"/>
 </resources>
 <connections/>
</ui>
